apworld/toontown/__init__.py

```diff
@@ -273,6 +273,16 @@ def fill_slot_data(self) -> Dict[str, Any]:
             if location.address and location.item and location.item.code and location.item.player == self.player
         ]
 
+        # If win condition is total_tasks, make sure that the player can actually complete them.
```

Comment:
Question: should we actually care if it's higher? Or should we just allow ourselves to require all tasks even if they're out of logic?

Reply:
for now we don't really care since that option doesn't actually restrict a player's ability to do tasks